tests/bootstrap.php

```diff
@@ -1,15 +1,33 @@
 <?php
+/**
```

Comment:
this file can be removed in favor of `./vendor/yiisoft/yii-core/tests/bootstrap.php` (should be updated in phpunit.xml.dist)